caffe2/python/operator_test/pack_ops_test.py

```diff
@@ -14,15 +14,19 @@
 
 class TestTensorPackOps(hu.HypothesisTestCase):
 
-    def pack_segments_ref(self, return_presence_mask=False):
-        def pack_segments_ref(lengths, data):
+    def pack_segments_ref(self, return_presence_mask=False, max_length=-1):
```

Comment:
nit: maybe max_length=None? In Python it's common to use None as default value to indicate optional argument.